go/internal/multipeer-connectivity-transport/driver/logger.gen.go

```diff
@@ -0,0 +1,9 @@
+// Code generated by berty.tech/core/.scripts/generate-logger.sh
```

Comment:
we're not using this method anymore, but I can help you get rid of this

Reply:
I've seen in other packages (e.g tinder) that logger is send from main.
For my package, I don't know how to do easily to give him the logger from main because it's a libp2p's Transporter constructor that constructs my transport.